cmd/provisioner-localpv/app/config.go

```diff
@@ -19,12 +19,13 @@ package app
 
 import (
 	//"fmt"
-	"path/filepath"
+	//"path/filepath"
 	"strings"
 
 	"github.com/golang/glog"
 	mconfig "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
 	cast "github.com/openebs/maya/pkg/castemplate/v1alpha1"
+	mHostPath "github.com/openebs/maya/pkg/hostpath/v1alpha1"
```

Comment:
the alias can be simply `hostpath` which make its look natural when someone reads the code

Reply:
done.